tests/conftest.py

```diff
@@ -30,7 +30,7 @@
 TOL = 1e-3
 CUTOFF = 6
 ALPHA = 0.1
-HBAR = 1.7
+HBAR = 2
```

Comment:
Is there a reason for using HBAR=2 in the tests? I would think a randomly picked (non-half-integer) value would be the best choice since it has the highest chance of flushing out bugs in the future.

Reply:
No reason, apart from `HBAR = 2` being the default for Strawberry Fields. I like this idea - do you think we could remove the `test_hbar` test? This test is one of the slower ones, due to using sampling.

Reply:
Yeah, the sampling takes a long time, I'll see if it could be sped up or avoided somehow.